src/ssl.c

```diff
@@ -5867,6 +5867,136 @@ int wolfSSL_CertManagerDisableCRL(WOLFSSL_CERT_MANAGER* cm)
 
     return WOLFSSL_SUCCESS;
 }
+
+void wolfSSL_CertManagerSetVerify(WOLFSSL_CERT_MANAGER* cm, VerifyCallback vc)
+{
+    WOLFSSL_ENTER("wolfSSL_CertManagerSetVerify");
+    if (cm == NULL)
+        return;
+
+    cm->verifyCallback = vc;
+}
+
+static int CertManager_DoVerifyCallback(WOLFSSL_CERT_MANAGER* cm, int ret,
```

Comment:
The code here is very similar to internal.c `DoVerifyCallback`. Did you consider combining any of that logic into a function both can use? I did a review and its possible, but the added logic for handling cm / ssl may not reduce the code size. Just something to consider.

Reply:
Yes, but as you mentioned, the extra handling involved with the SSL structure makes would make the code difficult to follow with little benefit.